code/modules/mob/living/brain/MMI.dm

```diff
@@ -111,6 +111,48 @@
 	else
 		return ..()
 
+/// Forces target brain into the MMI. Mainly intended for admin purposes, as this allows transfer without a mob or user.
+/obj/item/mmi/proc/force_brain_into(obj/item/organ/internal/brain/new_brain)
+	if(!new_brain)
```

Comment:
```suggestion
	if(isnull(new_brain))
```

prefer `isnull()` for typed references for explicitness

Reply:
also should this `stack_trace()`?

Reply:
Feexed, added a stack trace for this and for invalid type provided as new_brain param.